docs/modules/examples/pages/llama-parse-astra.adoc

```diff
@@ -135,7 +135,7 @@ print("\n***********New LlamaParse+ Basic Query Engine***********")
 print(response_1)
 ----
 +
-. Query the Astra vector store for an example with expected lack of context.
+. Query the Astra DB vector store for an example with expected lack of context.
```

Comment:
Should we specify Astra DB Serverless here?

Reply:
Yes.